akka-sse/src/main/scala/de/heikoseeberger/akkasse/EventStreamMarshalling.scala

```diff
@@ -38,7 +38,7 @@ object EventStreamMarshalling extends EventStreamMarshalling
 trait EventStreamMarshalling {
 
   implicit final def toResponseMarshaller(implicit ec: ExecutionContext): ToResponseMarshaller[Source[ServerSentEvent, Any]] =
-    Marshaller.withFixedCharset(MediaTypes.`text/event-stream`, HttpCharsets.`UTF-8`) { messages =>
+    Marshaller.withFixedContentType(ContentTypes.`text/event-stream`) { messages =>
```

Comment:
How would that even compile? `ContentTypes` is neither available as a simple identifier (imported), nor does it have a `text/event-stream` member. I'll have to check locally ...